From 05628dae987cd7d863122226ab951c87d180e167 Mon Sep 17 00:00:00 2001
Message-ID: <05628dae987cd7d863122226ab951c87d180e167.1694125609.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1694125609.git.sharlatanus@gmail.com>
References: <cover.1694125609.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 5 Sep 2023 23:40:37 +0100
Subject: [PATCH 02/19] gnu: libxisf: Extend description.

* gnu/packages/astronomy.scm (libxisf)[description]: Explain acronym.
Add essential links to the mentioned standard and the service.
---
 gnu/packages/astronomy.scm | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 624c17eade..278b82512c 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2810,8 +2810,11 @@ (define-public libxisf
     (home-page "https://nouspiro.space/?page_id=306")
     (synopsis "Astronomical library to load and write XISF file format")
     (description
-     "LibXISF is C++ library that can read and write XISF files produced by
-PixInsight.  It implements XISF 1.0 specification.")
+     "LibXISF is C++ library that can read and write @acronym{XISF,Extensible
+Image Serialization Format} files produced by @url{https://pixinsight.com/,
+PixInsight}.  It implements
+@url{https://pixinsight.com/doc/docs/XISF-1.0-spec/XISF-1.0-spec.html, XISF
+1.0 specification}.")
     (license license:gpl3+)))
 
 (define-public missfits
-- 
2.41.0

